### PR TITLE
Codex 10 heading

### LINKED ISSUE #10: Create Page Heading

### PR DESCRIPTION
**Pull request by Miran-Firdausi**
Closes #10 

Added Heading element
and added text to hero section
configured tailwind config to add gray color

### CODE REVIEW

File: src/assets/styles/index.css [hunk 1]
```diff
@@ -1,15 +1,24 @@
...
 }
 
 a {
   text-decoration: none;
   color: inherit;
 }
+
+.text-outlined {
+  -webkit-text-stroke-width: 2px;
+  -webkit-text-stroke-color: #737373;
```
Comment (Achyut-0705) [inline src/assets/styles/index.css:23]: Use tailwind classes as vars - check from inpect element --> html --> css variables

File: src/components/Heading/index.jsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import PropTypes from "prop-types";
+
+function Heading({ headingText }) {
```
Comment (Achyut-0705) [inline src/components/Heading/index.jsx]: Heading is a component, prop should be text

File: src/components/Heading/index.jsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import PropTypes from "prop-types";
+
+function Heading({ headingText }) {
+  return (
+    <div className="relative text-center font-black uppercase tracking-tighter">
+      <div className="xs:text-4xl sm:text-6xl lg:text-[6.5rem] text-outlined text-transparent">
+        {headingText}
```
Comment (Achyut-0705) [inline src/components/Heading/index.jsx]: Same here

File: src/components/Heading/index.jsx [hunk 4]
```diff
@@ -0,0 +1,20 @@
+import PropTypes from "prop-types";
+
+function Heading({ headingText }) {
+  return (
+    <div className="relative text-center font-black uppercase tracking-tighter">
+      <div className="xs:text-4xl sm:text-6xl lg:text-[6.5rem] text-outlined text-transparent">
+        {headingText}
+      </div>
+      <div className="xs:text-2xl sm:text-4xl lg:text-[3.75rem] text-foreground absolute w-full xs:bottom-[-10px] sm:bottom-[-18px] lg:bottom-[-12px]">
+        {headingText}
```
Comment (Achyut-0705) [inline src/components/Heading/index.jsx]: Here as well

File: src/components/Heading/index.jsx [hunk 5]
```diff
@@ -0,0 +1,20 @@
+import PropTypes from "prop-types";
+
+function Heading({ headingText }) {
+  return (
+    <div className="relative text-center font-black uppercase tracking-tighter">
+      <div className="xs:text-4xl sm:text-6xl lg:text-[6.5rem] text-outlined text-transparent">
+        {headingText}
+      </div>
+      <div className="xs:text-2xl sm:text-4xl lg:text-[3.75rem] text-foreground absolute w-full xs:bottom-[-10px] sm:bottom-[-18px] lg:bottom-[-12px]">
```
Comment (Achyut-0705) [inline src/components/Heading/index.jsx]: don't specify hard coded pixel values. instead use defaults provided by tailwind

File: src/pages/Home/index.jsx [hunk 6]
```diff
@@ -1,5 +1,7 @@
+import Hero from "./Hero";
+
 function Home() {
-  return <div>Home</div>;
+  return <Hero />;
```
Comment (Achyut-0705) [inline src/pages/Home/index.jsx]: Hero is a non functional component, replace it with actual jsx

File: tailwind.config.js [hunk 7]
```diff
@@ -4,12 +4,21 @@ export default {
...
     extend: {
       colors: {
-        background: '#232323',
-        primary: '#F24E3F',
-        text: '#F7F7F7',
+        background: "#232323",
+        primary: "#E76941",
+        foreground: "#F7F7F7",
+        gray: "737373",
+      },
```
Comment (Achyut-0705) [inline tailwind.config.js]: Use scheme as per object shared

primary: {
main: color
}

something like this

**Review by Achyut-0705 — CHANGES_REQUESTED** [review]
Make necessary changes

File: src/components/Heading/index.jsx (hunk 2 above)
Comment (Miran-Firdausi) [reply src/components/Heading/index.jsx]: This is done.

File: src/assets/styles/index.css (hunk 1 above)
Comment (Miran-Firdausi) [reply src/assets/styles/index.css:23]: I was not able to find the required tailwind variable.
The only thing i could find is those export variables are getting converted into this form.
image

Can you commit to this or help me figure out.

**Review by Achyut-0705 — APPROVED** [review]
LGTM!

File: src/assets/styles/index.css (hunk 1 above)
Comment (Achyut-0705) [reply src/assets/styles/index.css:23]: Will take care of this later

**Pull request merged**